### PR TITLE
New Payload: BitLockerKeyDump, WindowsLicenseKeyExfiltration -  New Extension: Windows_Fileless_HID_Exfil

### PR DESCRIPTION
**Pull request by 0iphor13**
- Uploaded payload for exfiltration of BitLocker recovery keys
- Uploaded new Extension for using Hak5s example of Keystroke Reflection, but fileless
- Uploaded Payload for exfiltration of Windows product keys

### CODE REVIEW

File: payloads/extensions/community/Windows_Fileless_HID_Exfil [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        USAGE:
+            Type out your command or script with powershell, afterwards you put the function Windows_Fileless_HID_Exfil() behind it.
+            It'll take the commands/scritps output and writes it into a variable, which then gets exfiltrated.
+
+        Example Usage:
+            GUI r
+            DELAY 500
+            STRINGLN powershell
+            DELAY 1000
+            STRING echo "This is my test"
```
Comment (dallaswinger) [inline payloads/extensions/community/Windows_Fileless_HID_Exfil:21]: important to note t(to the user) hat using `STRINGLN` here will likely break the pipe? (if i understand correctly)

aka any output that is required must not be fully executed prior to calling the Windows_Fileless_HID_Exfil function

File: payloads/extensions/community/Windows_Fileless_HID_Exfil [hunk 2]
```diff
@@ -0,0 +1,44 @@
+EXTENSION Windows_Fileless_HID_Exfil
```
Comment (dallaswinger) [inline payloads/extensions/community/Windows_Fileless_HID_Exfil:1]: a little late for me to mention that these names should be all caps to match the pattern used by the "official extensions"
-- i should have noticed earlier :(

File: payloads/extensions/community/Windows_Fileless_HID_Exfil (hunk 1 above)
Comment (0iphor13) [reply payloads/extensions/community/Windows_Fileless_HID_Exfil:21]: Indeed, will make it more clear

**Comment by dallaswinger** [comment]
LGTM - just needs that one "doc" update i mentioned

File: payloads/extensions/community/Windows_Fileless_HID_Exfil (hunk 2 above)
Comment (0iphor13) [reply payloads/extensions/community/Windows_Fileless_HID_Exfil:1]: :'o I try to update it and all the payloads using them in the next couple of days

**Comment by 0iphor13** [comment]
Changed all Extensions to being named in Capital letters and also updated payloads using them

**Comment by dallaswinger** [comment]
awesome ty, holding merge until I push a payloadstudio build out so that we dont mix them all up (in autocomplete and version checker etc)

**Review by dallaswinger — APPROVED** [review]
lgtm

**Pull request merged**